doc/template.html

```diff
@@ -54,4 +69,24 @@ <h2>Table of Contents</h2>
     </div>
   </div>
 </body>
+<script>
+  (function () {
```

Comment:
Why use a function here? That seems unnecessary. IIFE was nice and all in a ES5 world where `var` and function scope was the only thing, nowadays we should always prefer a code block instead.

```js
'use strict';
{
  const localVariable = 'some value';
}
typeof localVariable === 'undefined';
```

Reply:
@aduh95 Good point 👍 . 
I can make it a code block as well, but I think both serves the same purpose so I'll keep it to IIFE.

Reply:
If there is no reason to use IIFE, my objection remains: we should use a code block here. IMHO code blocks are more expressive, and it sends marginally less bytes to the end user.